hosting/locale/de/LC_MESSAGES/django.po

```diff
@@ -18,359 +18,273 @@ msgstr ""
-#: templates/hosting/login.html:17 templates/hosting/login.html.py:26
-#: templates/hosting/reset_password.html:32 templates/hosting/signup.html:30
-msgid "Login"
-msgstr "Anmelden"
-
-#: templates/hosting/bill_detail.html:11
 msgid "Invoice"
 msgstr "Rechnung"
 
-#: templates/hosting/bill_detail.html:11 templates/hosting/order_detail.html:23
 msgid "Order #"
 msgstr "Bestellung #"
 
-#: templates/hosting/bill_detail.html:25
 msgid "ungleich GmbH"
 msgstr ""
 
-#: templates/hosting/bill_detail.html:26
 msgid "buchhaltung@ungleich.ch"
 msgstr ""
 
-#: templates/hosting/bill_detail.html:27
 msgid "Hauptstrasse 14"
 msgstr ""
 
-#: templates/hosting/bill_detail.html:28
 msgid "CH-8775 Luchsingen"
 msgstr ""
 
-#: templates/hosting/bill_detail.html:29
 msgid "Mwst-Nummer: CHE-109.549.333 MWST"
 msgstr ""
 
-#: templates/hosting/bill_detail.html:60
 msgid "Total:"
 msgstr ""
 
-#: templates/hosting/bill_detail.html:68
 #, python-format
 msgid "Alles Preise in CHF mit 8%% Mehrwertsteuer."
 msgstr "All prices in CHF including 8%% VAT"
 
-#: templates/hosting/bill_detail.html:69
 msgid "Betrag zahlbar innerhalb von 30 Tagen ab Rechnungseingang."
 msgstr ""
 
-#: templates/hosting/bill_detail.html:70
 msgid "Kontoverbindung:"
 msgstr ""
 
-#: templates/hosting/bill_detail.html:73
 msgid "IBAN:"
 msgstr ""
 
-#: templates/hosting/bill_detail.html:76
 msgid "BIC:"
 msgstr ""
 
-#: templates/hosting/bill_detail.html:81
 msgid "CH02 0900 0000 6071 8848 8"
 msgstr ""
 
-#: templates/hosting/bill_detail.html:84
 msgid "POFICHBEXXX"
 msgstr ""
 
-#: templates/hosting/bills.html:12
 msgid "Customers"
 msgstr "Kunden"
 
-#: templates/hosting/bills.html:16 templates/hosting/user_keys.html.py:25
 msgid "Name"
 msgstr ""
 
-#: templates/hosting/bills.html:17
 msgid "Email"
 msgstr ""
 
-#: templates/hosting/bills.html:28
 msgid "View Bill"
 msgstr "Rechnung anzeigen"
 
-#: templates/hosting/bills.html:41 templates/hosting/orders.html.py:82
-#: templates/hosting/virtual_machines.html:70
 msgid "previous"
 msgstr "vorherige"
 
-#: templates/hosting/bills.html:47 templates/hosting/orders.html.py:88
-#: templates/hosting/virtual_machines.html:76
 msgid "next"
 msgstr "nächste"
 
-#: templates/hosting/confirm_reset_password.html:10
-#: templates/hosting/login.html:12 templates/hosting/reset_password.html:10
-#: templates/hosting/signup.html:9 templates/hosting/signup_validate.html:9
+msgid "SSH Key"
+msgstr "SSH Key"
+
+msgid "Choose a key option in order to access your VM"
+msgstr "Wähle eine Option um Zugriff auf deine VM zu erhalten"
+
+msgid "Generating a new key pair"
+msgstr "Neuen SSH-Key erstellen"
+
+msgid "I want to generate a new key pair"
+msgstr "Ich möchte einen neuen SSH-Key erstellen"
+
+msgid "Generate"
+msgstr "Generieren"
+
+msgid "Using existing key"
+msgstr "Nutzung eines existierenden SSH-Keys"
+
+msgid "I want to use my existing public key"
+msgstr "Ich möchte einen existierenden SSH-Key nutzen"
+
+#, fuzzy
```

Comment:
I would remove the fuzzy, and possibly confirm if "Upload" really translates to "Schlüssel hochladen". I guess "Schlüssel hochladen" is "Upload Key"